ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -383,8 +402,15 @@ class AvatarGroupDemoController: DemoTableViewController {
     }()
 
     @objc private func setMaxAvatarCount() {
-        if let text = maxAvatarsTextField.text, let count = Int(text) {
-            maxDisplayedAvatars = count
+        let oldMax = maxDisplayedAvatars
+
+        if let text = maxAvatarsTextField.text, let newMax = Int(text) {
```

Comment:
What happens if you cast the text to an int and the behavior is unexpected? Is it possible for that text to be anything but an integer?

Reply:
Since it's in an if, I think it should just skip that case. So if the text ends up failing to cast to int, then the newMax would be unchanged.